### ISSUE TITLE
Events are not fired anymore when adding recipient and sending mail fast

### ISSUE THREAD

**Issue by barisbikmaz** [OfficeDev/office-js#5093]
When sending multiple mails fast, after some time Outlook events are not fired anymore.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: Office on the web
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: ______
* Operating System: Windows
* Browser (if using Office on the web): Edge
 
## Expected behavior
Events should be fired

## Current behavior
When sending multiple mails fast after each other Outlook stops sending events, showing the Smart Alert Popup when event based is enabled.
After that, you cannot send any emails anymore because the code cannot complete the event.

You need to reload Outlook Online to send Emails again.

## Steps to reproduce
![event-problem-new-outlook](https://github.com/user-attachments/assets/96a4e233-9179-44b0-8d2e-c9edd0f3fbc3)

# Provide additional details
The code I am using for the runtime is simple and small:

```
const OnNewMessageCompose = (event) => {
     startRuntime(event, 'OnNewMessageCompose');
};

const OnMessageSend= (event) => {
    startRuntime(event, 'OnMessageSend');
};

const OnMessageRecipientsChanged = (event) => {
    startRuntime(event, 'OnMessageRecipientsChanged');
};

async function startRuntime(event, eventName){
  console.log(eventName);
  event.completed({ allowEvent: true });
}

```
## Context
We are providing a Mail Signature Solution for Outlook. This can cause issues on customer tenants.

**Comment by mobisw-msft** [comment]
Hey @barisbikmaz ,

Can you confirm if this is the same issue that you see as #4525 ?

**Comment by barisbikmaz** [comment]
@mobisw-msft Yes I can confirm. It seems to be same issue with one difference that also the OnMessageSend event stops also firing after a while.

**Comment by rkpathak** [comment]
@barisbikmaz The startRuntime function has an issue. Only for onSend, the allowEvent:true is meaningful. For other events this is not needed. Please separate event.completed for onSend from other events. Please test with this change.

**Comment by barisbikmaz** [comment]
@rkpathak Changed the code to only call "allowEvent: true" on MessageSend but the result was the same. Still having the issue after sending 10 - 15 mails.

**Comment by arpitsangwanmsft** [comment]
Hey @barisbikmaz! 
Yeah, we could repro this and it is related to an issue we already have in our backlog as mentioned in #4525.
Unfortunately, we have no timelines to share at this point.
 
Internal tracking ID: 4623248

**Comment by JuaneloJuanelo** [comment]
Team is working on the fix

**Comment by gostritm** [comment]
Hi @JuaneloJuanelo,
I wanted to check in and ask if there are any updates you can share. We’ve had customers reporting this issue, so we’re keen to know if there’s any progress or an estimated timeline for a fix.
Thanks for your time—I appreciate your efforts!

**Comment by JuaneloJuanelo** [comment]
Issue fixed bacj in July 15th, 2025. Closing this issue ,
